pkg/ibmcsidriver/node_helper_test.go

```diff
@@ -20,18 +20,87 @@
 package ibmcsidriver
 
 import (
+	"fmt"
 	"testing"
 
 	cloudProvider "github.com/IBM/ibmcloud-volume-file-vpc/pkg/ibmcloudprovider"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestProcessMount(t *testing.T) {
+	// test cases
+	testCases := []struct {
+		name              string
+		stagingTargetPath string
```

Comment:
are we really need staging, target and fs type all 3 parameter in file csi driver? looks not needed please check and update the PR accordingly

Reply:
@arahamad changed the staging variable to source to avoid confusion. Also fs type is mandatory parameter for mount command. Please check once